packages/ccd-js-gen/src/cli.ts

```diff
@@ -28,12 +30,33 @@ export async function main(): Promise<void> {
             '-o, --out-dir <directory>',
             'The output directory for the generated code'
         )
+        .option<lib.OutputOptions>(
+            '-t, --output-type [TypeScript|JavaScript|TypedJavaScript|Everything]',
+            'The output file types for the generated code.',
+            (value: string) => {
+                switch (value) {
+                    case 'TypeScript':
+                    case 'JavaScript':
+                    case 'TypedJavaScript':
+                    case 'Everything':
+                        return value as lib.OutputOptions;
```

Comment:
**Suggestion:** I think typescript is smart enough for us to avoid casting here.
```suggestion
                        return value;
```